sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -968,33 +963,42 @@ synchronized void storeRedirectionParameters(@NonNull String requestId, @NonNull
             mDistributeInfoTracker.updateDistributionGroupId(distributionGroupId);
             enqueueDistributionStartSessionLog();
             cancelPreviousTasks();
-            getLatestReleaseDetails(distributionGroupId, updateToken);
+            requestLatestReleaseDetails(distributionGroupId, updateToken);
         } else {
             AppCenterLog.warn(LOG_TAG, "Ignoring redirection parameters as requestId is invalid.");
         }
     }
 
     /**
-     * Get latest release details from server.
+     * Request latest release details from server.
      *
      * @param distributionGroupId distribution group id.
      * @param updateToken         token to secure API call.
      */
     @VisibleForTesting
-    synchronized void getLatestReleaseDetails(String distributionGroupId, String updateToken) {
-        AppCenterLog.debug(LOG_TAG, "Get latest release details...");
+    synchronized void requestLatestReleaseDetails(String distributionGroupId, final String updateToken) {
```

Comment:
Can you explain all these changes about the get API?

Reply:
I need to wrap the call into separate method to just call it in callback. Renaming `get` -> `request` is to avoid confusions because it's not a getter.